api/api_full.go

```diff
@@ -143,8 +153,8 @@ type FullNode interface {
 	MpoolGetNonce(context.Context, address.Address) (uint64, error)
 	MpoolSub(context.Context) (<-chan MpoolUpdate, error)
 
-	// MpoolEstimateGasPrice estimates what gas price should be used for a
-	// message to have high likelihood of inclusion in `nblocksincl` epochs.
+	// MpoolEstimateGasPrice is depracated
+	// Deprecated: use GasEstimateGasPrice instead
 	MpoolEstimateGasPrice(ctx context.Context, nblocksincl uint64, sender address.Address, gaslimit int64, tsk types.TipSetKey) (types.BigInt, error)
```

Comment:
Can probably just remove (in a followup PR)

Reply:
Yeah, I didn't want to bump API version just before we reset.